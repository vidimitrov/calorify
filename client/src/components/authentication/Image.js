import styled from 'styled-components';

const Image = styled.img`
  width: 85px;

  @media (max-width: 420px) {
    width: 65px;
  }
`;

export default Image;
