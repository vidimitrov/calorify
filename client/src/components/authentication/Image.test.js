import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Image from './Image';

it('Authentication/Image component renders correctly', () => {
  const tree = renderer.create(<Image src="../../assets/img/logo.png" />).toJSON();
  expect(tree).toMatchSnapshot();
});
