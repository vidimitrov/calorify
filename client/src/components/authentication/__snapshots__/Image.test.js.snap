// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/Image component renders correctly 1`] = `
.c0 {
  width: 85px;
}

@media (max-width:420px) {
  .c0 {
    width: 65px;
  }
}

<img
  className="c0"
  src="../../assets/img/logo.png"
/>
`;
